whylog/config/parser_name_manager.py

```diff
@@ -0,0 +1,45 @@
+import re
+
+
+class ParserNameManager(object):
+    WORDS_COUNT_IN_NAME = 2
+
+    def __init__(self, parsers):
+        self._parsers_name = set()
+        for parser_name in parsers:
+            self._parsers_name.add(parser_name)
+
+    def add_new_parser_name(self, parser_name):
+        self._parsers_name.add(parser_name)
+
+    def is_free_parser_name(self, parser_name, black_list):
```

Comment:
Hm, this method is used by private generator method. If removed this method we must know our config reference